body rendering (#921)
flame-engine/flame: packages/flame_forge2d/CHANGELOG.md

# CHANGELOG

> ## [0.8.1-releasecandidate.14]

Reviewer: This should just say `[Next]`

Author: done

>  - Behavior of BodyComponent render has changed so that it can work with flame coordinate system

Reviewer 2: ```suggestion
 - The rendering of `BodyComponent` is now inline with the Flame coordinate system
```
